### PR TITLE
{lib}[GCC/12.2.0] SentencePiece v0.1.99, gperftools v2.14

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3123.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/ebfd78cebf032554b89d533939bb68d5 for a full test report.

File: easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'gperftools'
+version = '2.14'
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb:4]: Hmm, we're still at 2.13 for GCCcore/13.2.0 and 2.12 for GCCcore/12.3.0, should we really use 2.14 for 12.2.0

Comment (boegel) [reply easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb:4]: Why would we not use latest `gperftools` version if it's not used yet in a particular generation of easyconfigs?

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb (hunk 1 above)
Comment (akesandgren) [reply easybuild/easyconfigs/g/gperftools/gperftools-2.14-GCCcore-12.2.0.eb:4]: For the sake of monotonically increasing versions :-)

**Pull request merged**